publish-gh-pages.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+git checkout master
+
+npm run build
+
+cp test/support/index.html dist/
+
+git checkout gh-pages
+
+mv dist/* .
+
+git add restricted-input.js
+git add index.html
+git commit -ve -m 'Update test page for restricted-input'
```

Comment:
- **v** to show verbose changes. We thought it would be nice to include the diff in the commit editor.
- **e** to open the editor allowing the developer to edit the commit message.